frameworks/CSharp/aspnetcore/Benchmarks/Data/RawDb.cs

```diff
@@ -64,6 +64,8 @@ DbCommand CreateReadCommand(DbConnection connection)
             id.Value = _random.Next(1, 10001);
             cmd.Parameters.Add(id);
 
+            (cmd as MySql.Data.MySqlClient.MySqlCommand)?.Prepare();
```

Comment:
@sebastienros Based on 14ad208cd437b09c0cbe6bbf9a58fed76e738cdf I decided to only opt MySqlConnector into this. It doesn't have npgsql's auto-prepare capability, so this improves performance by 10-15% (in my local Docker testing).

I tested changing this to just `cmd.Prepare()` with `aspcore-ado-pg` and got these results:

Test | Before | After | Change
--- | --- | --- | ---
Fortunes | 43492.60 | 43695.85 | -0.47%
DB | 46917.18 | 47240.47 | +0.69%

I'm not sure if this is significant, or just within the experimental margin-of-error.

Perhaps @roji has an opinion on whether explicit `Prepare()` should be used for both npgsql and MySqlConnector or left like this.

Reply:
Also we should probably update npgsql from 4.0.0-rc1 to 4.0.2, but I didn't address that in this PR.

Reply:
In principle an explicit `Prepare()` should be best for Npgsql. But IIRC there was some unexplained weirdness at some point where using auto-prepare was actually faster than explicit `Prepare()`. I didn't have any time to look into it (and don't have any right now either...), but maybe @sebastienros can re-confirm this...